corehq/apps/accounting/models.py

```diff
@@ -738,6 +738,10 @@ def at_max_domains(self):
             subscription_count += Subscription.visible_objects.filter(plan_version=version, is_active=True).count()
         return subscription_count >= self.max_domains
 
+    @property
+    def is_annual_plan(self):
+        return "Annual" in self.name
```

Comment:
This looks suspicious; logic based on string contents is fragile. Should at least lowercase both "Annual" and self.name.

Reply:
Yeah you're definitely right that this isn't a good way to be checking the software plans.

@nickpell do you know if there is a good way to tell if a plan that Julie makes is an annual one?

Reply:
(Offlined) Should add a column ```is_annual``` to the SoftwarePlan table.